lib/modules/sourceSnudown.js

```diff
@@ -40,6 +40,7 @@ function attachViewSourceButtons(entry) {
 		}
 
 		$menus::forEachChunked(item => {
+			if (item.querySelector('li.viewSource')) return; // guard against adding "source" button multiple times for whatever buggy reason
```

Comment:
I should probably reuse the `$(item).closest('li')` and check for `$that.siblings('.viewSource')`